### PR TITLE
OHARA-1584 Add quick start sample in README for develop streamApp

### PR DESCRIPTION
**Pull request by saivirtue**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,31 @@ An easy-to-use visual stream processing tool based on Apache Kafka.
 
 ### Getting Started
 
-[TODO]
+**Start Your Own Ohara StreamApp**
+
+For develop
```
Comment (konekoya) [inline README.md]: I'd rephrase this to *For development*

File: README.md [hunk 2]
```diff
@@ -4,7 +4,31 @@ An easy-to-use visual stream processing tool based on Apache Kafka.
...
+* you need to include the following jars into your project
```
Comment (konekoya) [inline README.md]: *in*?

File: README.md [hunk 3]
```diff
@@ -4,7 +4,31 @@ An easy-to-use visual stream processing tool based on Apache Kafka.
...
+```
+gradle clean jar -PskipManager
+cp ohara-common/build/libs/*.jar ohara-kafka/build/libs/*.jar ohara-streams/build/libs/*.jar  your_project
+```
+* sample ohara code can be seen in
```
Comment (konekoya) [inline README.md]: *found* sounds more natural to me

File: README.md [hunk 4]
```diff
@@ -4,7 +4,31 @@ An easy-to-use visual stream processing tool based on Apache Kafka.
...
+```
+# for external cluster
+ohara-streams/src/test/java/com/island/ohara/streams/SimpleApplicationForExternalEnv.java
+# for ohara environment
+ohara-streams/src/test/java/com/island/ohara/streams/SimpleApplicationForOharaEnv.java
+```
+
+For compile
```
Comment (konekoya) [inline README.md]: Maybe *compilation*

File: README.md [hunk 5]
```diff
@@ -4,7 +4,31 @@ An easy-to-use visual stream processing tool based on Apache Kafka.
...
+* beside ohara libraries, you will also need to include kafka libraries
```
Comment (konekoya) [inline README.md]: *In addition to ohara librarise*

File: README.md [hunk 6]
```diff
@@ -24,7 +48,6 @@ gradle licenseApply
 ### Installing
 
 **Running Ohara In Docker**
-- - - 
 * Ensure your nodes (actual machine or VM) are installed Docker 18.09+
```
Comment (konekoya) [inline README.md]: *Have installed*?

**Comment by saivirtue** [comment]
> Some suggestions on wordings :)

Thanks a ton !
